users/faq.rst

```diff
@@ -494,3 +494,31 @@ to do this.  The most well known is called daemontools, and can be found in the
 standard package repositories for  almost every modern Linux distribution.
 Other popular tools with similar functionality include S6 and the aforementioned
 runit.
+
+.. _inotify-limits:
+
+How do I increase the inotify limit to get my fs watcher to work?
+-----------------------------------------------------------------
+
+You are probably reading this because you encountered the following error with
+the fs watcher on linux:
+
+    Failed to start filesystem watcher for folder yourLabel (yourID): failed to
+    setup inotify handler. Please increase inotify limits, see
+    https://docs.syncthing.net/users/faq.html#inotify-limits
+
+Linux typically restricts the amount of watches per user (usually 8192). So if
+you have a more directories, you need to adjust that number.
+
+On many Linux distributions you can run the following to fix it: ::
+
+    echo "fs.inotify.max_user_watches=204800" | sudo tee -a /etc/sysctl.conf
+
+On Arch Linux and potentially others it is preferred to write this line into a
+separate file, i.e. you should run ::
```

Comment:
Here, and above/below, you can end the line with just the double colon to both show a normal colon and make the following paragraph a code block. That is,

`... you should run::`

becomes

<img width="211" alt="screen shot 2018-03-23 at 13 27 20" src="https://user-images.githubusercontent.com/125426/37829097-f6088390-2e9d-11e8-8691-f692c8562e1d.png">